main.go

```diff
@@ -4,28 +4,51 @@ import (
 	// Import the Discordgo package, and other required packages.
```

Comment:
It looks like you missed out on import statements!

Reply:
Wtf i had it when I was coding it should be fixed now tho

Reply:
I think you missed another import `fmt`